modules/cnvkit/1.0/config/default.yaml

```diff
@@ -0,0 +1,64 @@
+lcr-modules:
+    
+    cnvkit:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            sample_bai: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
```

Comment:
Can you please add in-line documentation for these options? What do they mean/how to use them?

Reply:
Done